doc/md/fundamentals/1-basic-syntax/4-numbers.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 4
+---
+
+# Numbers
+
+## Integers
+
+[`Int`](https://internetcomputer.org/docs/motoko/base/Int) represents all integers, both positive and negative (e.g., -2, -1, 0, 1, 2).
+
+For scenarios requiring fixed-size integers, Motoko offers bounded variants with specific bit-widths ([`Int8`](https://internetcomputer.org/docs/motoko/base/Int8), [`Int16`](https://internetcomputer.org/docs/motoko/base/Int16), [`Int32`](https://internetcomputer.org/docs/motoko/base/Int32), [`Int64`](https://internetcomputer.org/docs/motoko/base/Int64)). These types can overflow if their limits are exceeded, resulting in a [runtime error](https://internetcomputer.org/docs/motoko/fundamentals/error-handling).
+
+```motoko no-repl
+let a : Int = -42;
+let b : Int = 0;
+let c : Int = 12345;
+```
+
+### Unbounded integers
+
+By default, [`Int`](https://internetcomputer.org/docs/motoko/base/Int) is unbounded, meaning it can grow as large (or as small) as needed without causing overflow.
+
+```motoko no-repl
+let bigNumber : Int = 999_999_999_999_999;
+```
+
+### Bounded integers
+
+For scenarios requiring fixed-size integers, Motoko offers bounded integer types with specific bit-widths.
+
+- [`Int8`](https://internetcomputer.org/docs/motoko/base/Int8)  (8-bit signed integer)
+- [`Int16`](https://internetcomputer.org/docs/motoko/base/Int16) (16-bit signed integer)
+- [`Int32`](https://internetcomputer.org/docs/motoko/base/Int32) (32-bit signed integer)
+- [`Int64`](https://internetcomputer.org/docs/motoko/base/Int64) (64-bit signed integer)
+
+Bounded integers can overflow if their limits are exceeded, resulting in a [runtime error](https://internetcomputer.org/docs/motoko/fundamentals/error-handling).
+
+```motoko
+let overflowInt : Int8 = 128; // Error: literal out of range Int8
+```
+
+
+## Natural numbers
+
+The [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) type represents natural numbers, which are all non-negative integers (i.e., `0` and positive numbers).
```

Comment:
maybe `Nat` first, before integers? This is the most common number so it probably should be the first